game/menu/gamemanager.gd

```diff
@@ -39,37 +190,36 @@ func _build_menu():
 	_main.show()
 
 	#making the buttons
-	for game in _games:
+	for game_id in GAMES.keys():
 		var display = _preview_scene.instance()
-		display.setup(game)
+		display.setup(GAMES[game_id])
 		display.connect("pressed", self, "load_game")
 		_grid.add_child(display)
+		GAME_DISPLAYS[game_id] = display
 
 
 # go through every folder inside res://games/ and try to load the game.cfg into _games
 func _find_games():
```

Comment:
The Comment is outdated anyway. It isn't called _games for a few days now.
Still, I think it fits because a game is "found" when it is in the game list.
So finding all games requires writing them into the game list.

Reply:
I don't know how to split it, and again, this wasn't made by me. So I just renamed it to _find_and_load_games